Say Their Names/Source/View/Home/Person/PersonPhotoCell.swift

```diff
@@ -8,19 +8,57 @@
 
 import UIKit
 
-class PersonPhotoCell: UICollectionViewCell {
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+final class PersonPhotoCell: UICollectionViewCell {
+    
+    private let personImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        personImageView.image = nil
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setup() {
+        addSubview(personImageView)
+                
+        let constraints = [
+            personImageView.topAnchor.constraint(equalTo: topAnchor),
+            personImageView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            personImageView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            personImageView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ]
+        
```

Comment:
Added a handy extension. (UIView++)
I'll change line 39-42 to `personImageView.fillSuperview()`